3Dmol/glmodel.js

```diff
@@ -2447,34 +2457,40 @@ $3Dmol.GLModel = (function() {
 
     /**
     * Set coordinates for the atoms parsed from various topology files. 
-    * @function $3Dmol.GLModel#setCoordinatesFromUrl
+    * @function $3Dmol.GLModel#setupFrames
     * @param {string} url - contains the url where mdsrv has been hosted
     * @param {string} path - contains the path of the file (<root>/filename)
-    * @param {object} viewer - contains the glviewer object
     */
 
-        this.setCoordinatesFromUrl = function(url, path) {
+        this.setupFrames = function (url, path) {
```

Comment:
I prefer the old name.  If data is being hosted remotely, I think we should assume it likely won't fit in memory and we should pre-create frames.  Instead they should be created dynamically, on-the-fly.  This should do the minimum amount of work to provide the information to make setFrame work.